### PR TITLE
Conode runscript 721

### LINKED ISSUE #721: cothorityd-runscript
create a script for cothorityd so that it gets restarted even if the daemon dies. Perhaps add an option to update the packet.

### PR DESCRIPTION
**Pull request by ineiti**
Closes #721

### CODE REVIEW

File: run_conode.sh [hunk 1]
```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env bash
+set -e
+
+VERSION=1.0-pre1
+MAILCMD=mail
+MAILADDR=linus.gasser@epfl.ch
```
Comment (nikkolasg) [inline run_conode.sh]: Do you have any reference as how to use run_conode ? in the README ? 
If so, do you specifiy that it sends a log to this address automatically if mail is installed ?

File: run_conode.sh [hunk 2]
```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env bash
+set -e
+
+VERSION=1.0-pre1
```
Comment (nikkolasg) [inline run_conode.sh]: Well, we should have a consistent versionning system, putting at two places seems like trouble for the future...

Comment (nikkolasg) [inline run_conode.sh]: I guess it's possible to make onet.Server prints out the version or stg ? Or conode binary ?

Comment (ineiti) [inline run_conode.sh]: hmm - can't run onet.Server here... But I can do some grep-magick...

File: run_conode.sh [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+	exec $RUN_CONODE "$ALL_ARGS"
+}
+
+migrate(){
+	PATH_VERSION=$PATH_CO/conode/version
+	if [ -d $PATH_CO/cothorityd ]; then
+		echo "Moving cothorityd-directory to conode"
+		mv $PATH_CO/cothorityd $PATH_CO/conode
+		echo 0.9 > $PATH_VERSION
+	elif [ -d $PATH_CO/cothority ]; then
```
Comment (nikkolasg) [inline run_conode.sh]: is it not the same thing ?

File: run_conode.sh (hunk 1 above)
Comment (ineiti) [inline run_conode.sh]: Only when called with `-mail`. I'll add that to the output.

File: run_conode.sh (hunk 3 above)
Comment (ineiti) [inline run_conode.sh]: no, there has been some `cothorityd` and some `cothority`...

**Comment by nikkolasg** [comment]
I don't quite get the difference between `run_conode.sh <public> <private>` and just `cd conode; go build && ./conode`. 
In general, can you add a very short sentence to say what is run_conode.sh useful for and how to use it ?

**Comment by ineiti** [comment]
`run_conode.sh` gives you:
- migration of data in {.config,Library}/{cothority,conode}
- loop around `conode` if it dies
- automatic updates (-update)
- sending of log-files (-mail)

File: conode/README.md [hunk 4]
```diff
@@ -72,6 +72,22 @@ To start your conode with the default (private) configuration file, located at `
...
+* migration of data if we change it
+* loop around conode if it quits
+* automatic updates (-update)
+* sending of log-files (-mail)
+
+You can run it with all extras like so:
+
+`./run_conode.sh -mail -update`
+
+And it will send an email to DEDIS with the last 200 lines of log-output every time conode restarts.
```
Comment (nikkolasg) [inline conode/README.md]: Without the `And`.

**Pull request merged**